playbooks/99-logs.yml

```diff
@@ -42,36 +46,40 @@
       ansible.builtin.import_role:
         name: env_op_images
 
-    - name: Ensure ansible.log is at the expected location
-      register: ansible_log_state
-      ansible.builtin.stat:
-        path: "{{ ansible_user_dir }}/ansible.log"
+    - name: Create a versioned log folder
+      ansible.builtin.file:
+        path: "{{ logfiles_dest_dir }}"
+        state: directory
+        mode: "0775"
+
+    - name: Return a list of log files in home directory
+      ansible.builtin.find:
+        paths: "{{ ansible_user_dir }}"
+        patterns: '*.log'
+      register: _log_files
 
     - name: Ensure ansible facts cache exists
       register: ansible_facts_cache_state
       ansible.builtin.stat:
         path: "{{ ansible_user_dir }}/ansible_facts_cache"
 
-    - name: Generate log file name date
-      register: filename_date
-      ansible.builtin.command:
-        cmd: date +%F-%R
-
-    - name: Copy ansible log if exists
+    - name: Copy log files
       when:
-        - ansible_log_state.stat.exists is defined
-        - ansible_log_state.stat.exists | bool
+        - _log_files.matched > 0
       block:
-        - name: Copy ansible log to proper location
+        - name: Copy logs to proper location
           ansible.builtin.copy:
-            src: "{{ ansible_user_dir }}/ansible.log"
-            dest: "{{ cifmw_basedir|default(ansible_user_dir ~ '/ci-framework-data') }}/logs/ansible.log-{{ filename_date.stdout|trim }}"
+            src: "{{ item.path }}"
+            dest: "{{ logfiles_dest_dir + item.path | basename }}"
```

Comment:
A `/` may be missing between both parts of the concatenation, as the `logfiles_dest_dir` ends with the timestamp and not with / . I'd prefer to see this joined using path_join.

Reply:
:100:

Reply:
ack!